### PR TITLE
Update information on the translation project

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Update information on the translation project.

 

### Details

- None

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -180,13 +180,15 @@ There is a `pyvista translation page`_ for pyvista (main) documentation.
...
+7. Translation is backuped in `pyvista-doc-translations`.
 
 Details can be found here: https://docs.transifex.com/getting-started-1/translators
 
-.. _`pyvista translation page`: https://www.transifex.com/getfem-doc/pyvista-doc/
+.. _`pyvista translation page`: https://www.transifex.com/tkoyama010/pyvista-doc/
 .. _Transifex: https://www.transifex.com/
-.. _`Read The Docs`: https://pyvista-doc.readthedocs.io/en/latest
+.. _`GitHub Pages`: https://github.com/tkoyama010/pyvista-docs-dev-ja
+.. _`GitHub repository`: https://github.com/tkoyama010/pyvista-docs-dev-ja
```
Comment (tkoyama010) [inline doc/index.rst:191]:
```suggestion
.. _`GitHub Pages`: https://github.com/tkoyama010/pyvista-docs-dev-ja
.. _`GitHub repository`: https://github.com/tkoyama010/pyvista-docs-dev-ja
.. _`pyvista-doc-translations`: https://github.com/tkoyama010/pyvista-doc-translations
```

File: doc/index.rst [hunk 2]
```diff
@@ -180,13 +180,16 @@ There is a `pyvista translation page`_ for pyvista (main) documentation.
...
+.. _`pyvista-doc-translations`: https://github.com/tkoyama010/pyvista-doc-translations
```
[the remark this answers is not in the document]
Comment (tkoyama010) [reply doc/index.rst:192]:
```suggestion
.. _`GitHub Pages`: https://pyvista.github.io/pyvista-docs-dev-ja/index.html
.. _`GitHub repository`: https://github.com/pyvista/pyvista-docs-dev-ja
.. _`pyvista-doc-translations`: https://github.com/pyvista/pyvista-doc-translations
```

File: doc/index.rst [hunk 3]
```diff
@@ -180,13 +180,16 @@ There is a `pyvista translation page`_ for pyvista (main) documentation.
 3. Click ``Request language`` and fill form.
 4. Wait acceptance by transifex pyvista translation maintainers.
 5. (After acceptance) Translate on transifex.
-6. You can see the translated document in `Read The Docs`_.
+6. You can host the translated document in `GitHub Pages`_ by creating `GitHub repository`.
```
Comment (tkoyama010) [inline doc/index.rst:183]:
```suggestion
6. We can host the translated document in `GitHub Pages`_ by creating `GitHub repository`.
```

**Review by akaszynski — APPROVED** [review]
LGTM

File: doc/index.rst [hunk 4]
```diff
@@ -180,13 +180,16 @@ There is a `pyvista translation page`_ for pyvista (main) documentation.
 3. Click ``Request language`` and fill form.
 4. Wait acceptance by transifex pyvista translation maintainers.
 5. (After acceptance) Translate on transifex.
-6. You can see the translated document in `Read The Docs`_.
+6. We can host the translated document in `GitHub Pages`_ by creating `GitHub repository`.
```
Comment (adeak) [inline doc/index.rst:183]: Should this be a link?
```suggestion
6. We can host the translated document in `GitHub Pages`_ by creating `GitHub repository`_.
```

Comment (adeak) [reply doc/index.rst:183]: It has to be; let me commit it.

**Pull request merged**